app/uk/gov/hmrc/leakdetection/services/SummaryService.scala

```diff
@@ -66,7 +66,7 @@ class SummaryService @Inject()(ruleService: RuleService,
             filteredLeaks.filter(l => l.repoName == r),
             filteredWarnings.filter(w => w.repoName == r),
             filteredBranches.filter(a => a.repoName == r),
-            if(archivedRepos.contains(r)) true else false)
+            archivedRepos.contains(r))
```

Comment:
It looks like the intermediate tuple (which are unlabelled and easy to make mistakes) can be avoided by mapping directly to `RepositorySummary`? Named params would be helpful too.